Source/Core/Core/HW/WiimoteReal/IONix.cpp

```diff
@@ -15,6 +15,7 @@
 
 namespace WiimoteReal
 {
+extern Wiimote* g_wiimotes[MAX_BBMOTES];
```

Comment:
(Preferably: redesign the API so the dependency on global state is not required.)